demo/README.md

```diff
@@ -13,9 +13,46 @@ to an FPC network, a FPC network and a chaincode to execute the auction logic.
 
 ## Bring Up the Demo End To End
 ### Setup
-To set up all components at once run the start script.
+To set up all components at once run the start script. An [FPC Network](../utils/docker-compose/network-config/docker-compose.yml).
 ```
-scripts/startFPCAuctionNetwork.sh
+cd $FPC_PATH/demo
+scripts/startFPCAuctionNetwork.sh --build-cc
+```
+
+The channel `mychannel` will be created and used to install and instantiate
+all chaincodes. The [golang mock chaincode](chaincode/golang) will be
+instantiated as `mockcc` and the [FPC auction chaincode](chaincode/fpc) will be instantiated as `auctioncc`. If you do not need to build the FPC Auction CC,
```

Comment:
You might mention also that the fabric-gateway is configured for the fpc chaincode auctioncc and point to the config.json if you want to change that?